AttributeError: 'CachedResponse' object has no attribute 'closed'

Hi, I am trying to use `aiohttp-client-cache` in combination with `aiohttp-retry`.

```python
retry_options = ExponentialRetry(attempts=10)
async with CachedSession(cache=SQLiteBackend('demo_cache')) as session:
  retry_client = RetryClient(client_session=session, raise_for_status=False, retry_options=retry_options)
  async with retry_client.get(some_url, params=some_params, compress=True) as response:
    print(response)
```

### The problem
`retry_client.get` checks `response.closed` which is not defined for `CachedResponse` leading to an error in [aiohttp-retry](https://github.com/inyutin/aiohttp_retry/blob/ba2169891f5b32a5c59e48ca185dd8e68e44ded7/aiohttp_retry/client.py#L159).

```
exc_type = None, exc_val = None, exc_tb = None

    async def __aexit__(
        self,
        exc_type: Optional[Type[BaseException]],
        exc_val: Optional[BaseException],
        exc_tb: Optional[TracebackType],
    ) -> None:
        if self._response is not None:
>           if not self._response.closed:
E           AttributeError: 'CachedResponse' object has no attribute 'closed'

/opt/homebrew/lib/python3.11/site-packages/aiohttp_retry/client.py:158: AttributeError
```

### Expected behavior
The `closed` attribute is defined similar as in [aiohttp](https://github.com/aio-libs/aiohttp/blob/19ffc6453a5c44ed09234ff8bd0ce8092f99e87d/aiohttp/client_reqrep.py#L924). Always returning `True` seems like a valid solution to me.

### Workarounds
Monkey patching `response.closed=True` works and retry functionality is working afterwards.

### Environment
- aiohttp-client-cache version: `0.9.0`
- Python version: `3.11.5`

--- a/aiohttp_client_cache/response.py
+++ b/aiohttp_client_cache/response.py
@@ -232,7 +232,7 @@ class CachedResponse(HeadersMixin):
 
     async def __aenter__(self) -> 'CachedResponse':
         return self
-    
+
     @property
     def closed(self) -> bool:
         return True

--- a/aiohttp_client_cache/response.py
+++ b/aiohttp_client_cache/response.py
@@ -232,6 +232,10 @@ class CachedResponse(HeadersMixin):
 
     async def __aenter__(self) -> 'CachedResponse':
         return self
+    
+    @property
+    def closed(self) -> bool:
+        return True
 
     def close(self):
         pass

--- a/test/unit/test_response.py
+++ b/test/unit/test_response.py
@@ -196,6 +196,7 @@ async def test_no_ops(aiohttp_client):
     await response.start()
     response.release()
     response.close()
+    assert response.closed is True
     await response.wait_for_close()
     await response.terminate()
     assert response.connection is None

